hardhat/.gitignore

```diff
@@ -0,0 +1,19 @@
+node_modules
```

Comment:
* Please do not add build artefacts as well as deployed contract addresses to the repository. The `hardhat/build` path should be ignored.
* Also I propose to avoid using tech-specific names for directories. Instead of `truffle` or `hardhat` the folder should be called `blockchain` or `backend` or something like that.